src/components/MapLibreMap.tsx

```diff
@@ -1,22 +1,34 @@
+import type { MapLayerMouseEvent } from 'maplibre-gl';
 import 'maplibre-gl/dist/maplibre-gl.css';
 import { RMap } from 'maplibre-react-components';
+import { getHoydeFromPunkt } from '../api/getHoydeFromPunkt';
+import { useEffect, useState } from 'react';
 
-const mountain: [number, number] = [6.4546, 46.1067];
 
-function MapLibreMap() {
-  return (
-    <div style={{ position: 'relative', height: '400px', width: '100%' }}>
-      <RMap
-        minZoom={6}
-        initialCenter={mountain}
-        initialZoom={8}
-        mapStyle="https://openmaptiles.geo.data.gouv.fr/styles/osm-bright/style.json"
-        style={{ height: '100%', width: '100%' }}
-      />
-    </div>
-  );
-}
+const trondheim: [number, number] = [10.40565401, 63.4156575];
 
+export const MapLibreMap = () => {
+    const [hoyde, setHoydeAtPunkt] = useState<undefined | number>(undefined);
 
+    useEffect(() => {
+        console.log(hoyde);
+    }, [hoyde]);
 
-export default MapLibreMap;
+    const onMapClick = async (e: MapLayerMouseEvent) => {
+        const hoyder = await getHoydeFromPunkt(e.lngLat.lng, e.lngLat.lat);
+        setHoydeAtPunkt(hoyder[0].Z)
+    }
+
+    return (
+        <div style={{ position: 'relative', height: '1000px', width: '100%' }}>
```

Comment:
```suggestion
        <div style={{ position: 'relative', height: '80vh', width: '100%' }}>
```
1000px er fort større enn viewports på laptoper